packages/core/src/web/app/actions/beambox/font-funcs.ts

```diff
@@ -682,51 +686,93 @@ const convertTextToPath = async (
+          d,
+          id: svgCanvas.getNextId(),
+          'stroke-dasharray': 'none',
+          'stroke-opacity': '1',
+          transform,
+          'vector-effect': 'non-scaling-stroke',
+        });
 
-      if (textAttr['stroke-width']) {
-        path.setAttribute('stroke-width', (+textAttr['stroke-width'] / 2).toString());
-      }
+        if (textAttr['stroke-width']) {
+          path.setAttribute('stroke-width', (+textAttr['stroke-width'] / 2).toString());
+        }
 
-      textElement.parentNode!.insertBefore(path, textElement.nextSibling);
-      path.addEventListener('mouseover', svgCanvas.handleGenerateSensorArea);
-      path.addEventListener('mouseleave', svgCanvas.handleGenerateSensorArea);
-      svgCanvas.pathActions.fixEnd(path);
-      batchCmd.addSubCommand(new history.InsertElementCommand(path));
+        textElement.parentNode!.insertBefore(path, textElement.nextSibling);
+        path.addEventListener('mouseover', svgCanvas.handleGenerateSensorArea);
+        path.addEventListener('mouseleave', svgCanvas.handleGenerateSensorArea);
+        svgCanvas.pathActions.fixEnd(path);
+        batchCmd.addSubCommand(new history.InsertElementCommand(path));
 
-      if (moveElement) {
-        // output of fluxsvg will locate at (0,0), so move it.
-        moveElements([moveElement.x], [moveElement.y], [path], false);
-      }
+        if (moveElement) {
+          // output of fluxsvg will locate at (0,0), so move it.
+          moveElements([moveElement.x], [moveElement.y], [path], false);
+        }
+
+        svgedit.recalculate.recalculateDimensions(path);
+      } else {
+        const group = document.createElementNS(svgedit.NS.SVG, 'g');
+
+        textElement.parentNode!.insertBefore(group, textElement.nextSibling);
+        group.addEventListener('mouseover', svgCanvas.handleGenerateSensorArea);
+        group.addEventListener('mouseleave', svgCanvas.handleGenerateSensorArea);
+        batchCmd.addSubCommand(new history.InsertElementCommand(group));
+
+        d.forEach((dStr) => {
+          const path = document.createElementNS(svgedit.NS.SVG, 'path') as unknown as SVGPathElement;
+
+          setAttributes(path, {
+            ...textAttr,
+            d: dStr,
+            id: svgCanvas.getNextId(),
+            'stroke-dasharray': 'none',
+            'stroke-opacity': '1',
+            transform,
+            'vector-effect': 'non-scaling-stroke',
+          });
+
+          if (textAttr['stroke-width']) {
+            path.setAttribute('stroke-width', (+textAttr['stroke-width'] / 2).toString());
+          }
 
-      svgedit.recalculate.recalculateDimensions(path);
+          group.appendChild(path);
+          path.addEventListener('mouseover', svgCanvas.handleGenerateSensorArea);
+          path.addEventListener('mouseleave', svgCanvas.handleGenerateSensorArea);
+          svgCanvas.pathActions.fixEnd(path);
+          batchCmd.addSubCommand(new history.InsertElementCommand(path));
```

Comment:
nitpick:
`InsertElementCommand` of path could be omitted because group is also newly added to reduce command size